test/JDBC/expected/babel_datatype_sqlvariant-vu-prepare.out

```diff
@@ -753,3 +753,86 @@ insert into babel_datatype_sqlvariant_vu_prepare_t9 values (cast('$200' as money
 go
 ~~ROW COUNT: 1~~
 
+
+create table babel_4036_t1( a int, b sql_variant, c varchar(50), d sql_variant, e sql_variant);
+go
+
+insert into babel_4036_t1(a,c,e) values (1, 'String 1', cast('Varchar String' as varchar(50)));
+go
+~~ROW COUNT: 1~~
+
+
+insert into babel_4036_t1 values (2, cast('😊😋😎😍😅😆' as nchar(24)), 'String 2', cast('😊😋😎😍😅😆' as nchar(26)),cast('😊  😋 😎dsfsd😍 😅ds😆' as nchar(24)));
```

Comment:
we should have test case for each type we are handling, char, nvarchar, binary, varbinary, money bit and so on